docs/use-cases/organizational-access-control.mdx

```diff
@@ -0,0 +1,38 @@
+---
+
+<!--todo: description --->
+
+---
+
+# Organizational Access Control
+
+## Abstract
+
+Traditional organizations mainly have a distribution of access to resources based on the roles.
+DAOs often do not have such a mechanism, but both types of organizations suffer from similar problems, mainly related to privacy.
+The most obvious ones are:
```

Comment:
Some of these points are better suited for voting use case, rather than access control